plugins/gateway/kong/README.md

```diff
@@ -1,5 +1,75 @@
 ## APIClarity Kong plugin
 
+### Prerequisite
+* APIClarity backend is running
+* Kong gateway is running in your K8s cluster, and configured correctly.
+
+If you just want to experience with it, and you don't have kong, you can take these few quick steps that will help you to quickly setup a running environment:
```

Comment:
If you just want to try it out, and you don't have kong installed, you can follow these few short steps that will help you to quickly setup a running environment: